independent-projects/resteasy-reactive/server/runtime/src/main/java/org/jboss/resteasy/reactive/server/core/multipart/MultipartSupport.java

```diff
@@ -55,27 +56,59 @@ public static MultipartFormDataInput toMultipartFormDataInput(ResteasyReactiveRe
         return formData.toMultipartFormDataInput();
     }
 
-    @SuppressWarnings({ "unchecked", "rawtypes" })
-    public static Object convertFormAttribute(String value, Class type, Type genericType, MediaType mediaType,
-            ResteasyReactiveRequestContext context, String attributeName) {
+    public static Object getConvertedFormAttribute(String attributeName, Class type, Type genericType, MediaType mediaType,
```

Comment:
Let's please remove lambdas from the code

Reply:
Yup, fair. I found 40+ implementations of a constant `Supplier` in our code base but none of them directly accessible :(